apps/verticalhome/js/app.js

```diff
@@ -86,6 +88,16 @@
       }
     },
 
+    isinPinnedApp: function(result) {
```

Comment:
Creation of at least brief description would be really nice for new functions, because function's purpose isn't always clear just from it's name